binders/rabbit-binder/spring-cloud-stream-binder-rabbit/src/test/java/org/springframework/cloud/stream/binder/rabbit/LocalizedQueueConnectionFactoryIntegrationTests.java

```diff
@@ -34,20 +35,22 @@
 
 /**
  * @author Gary Russell
+ * @author Chris Bono
  */
 public class LocalizedQueueConnectionFactoryIntegrationTests {
 
+	private static final RabbitMQContainer RABBITMQ = RabbitTestContainer.sharedInstance();
+
 	@RegisterExtension
-	public static RabbitTestSupport rabbitAvailableRule = new RabbitTestSupport(true);
```

Comment:
I moved all `RabbitTestSupport` instance variables to private non-final scope. There is no need for public. There is no big win for `static final` and it actually makes one think that it has to be final to prevent Rabbit from starting every test method - but that is not the case and that is controlled in `RabbitTestContainer`. 

Based on the above I made this consistent in all of these tests.